identity/Makefile

```diff
@@ -1,7 +1,3 @@
-IDEN3_DIR = $(HOME)/iden3
```

Comment:
Should keep `IDEN3_DIR` -- it's still used.

Reply:
Could change the Makefile to use `IDEN3_WORKDIR` instead, for consistency. Or vice-versa.